.travis.yml

```diff
@@ -1,39 +1,4 @@
-language: php
-dist: trusty
+version: ~> 1.0
 
-matrix:
-  include:
-    - php: 5.6
-      env: DB=MYSQL INSTALLER_VERSION=4.2.x-dev PHPCS_TEST=1 PHPUNIT_TEST=1
-    - php: 7.0
-      env: DB=PGSQL INSTALLER_VERSION=4.2.x-dev PHPUNIT_TEST=1
-    - php: 7.1
-      env: DB=MYSQL INSTALLER_VERSION=4.3.x-dev PHPUNIT_COVERAGE_TEST=1
-    - php: 7.2
-      env: DB=MYSQL INSTALLER_VERSION=4.3.x-dev PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.4.x-dev PHPUNIT_TEST=1 NPM_TEST=1
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.5.x-dev PHPUNIT_TEST=1
-    - php: 7.3
-      env: DB=MYSQL INSTALLER_VERSION=4.x-dev PHPUNIT_TEST=1
-
-before_script:
-  - phpenv rehash
-  - phpenv config-rm xdebug.ini
-  - composer validate
-  - composer require silverstripe/installer:"$INSTALLER_VERSION" ezyang/htmlpurifier:* --no-update
```

Comment:
> ezyang/htmlpurifier:*

that package is missing. do we still need it?

Reply:
I've added it in under env global REQUIRE_EXTRA